### ISSUE TITLE
Support Web

### ISSUE THREAD

**Issue by kuhnroyal** [luckyseven/avatars#10]
Currently the project pulls sqflite, this is pretty unexpected.
Could probably work with a datastore that is web capable.

**Comment by marcoancona** [comment]
Agree, `sqflite` seems quite an overkill. Could probably use something more lightweight.

**Comment by luckyseven** [comment]
thanks for your feedback! I'll look for an alternative soon. Also, i'm going to release a new package version to add full-support for web (the main issue now is the cache system, but it's more a "time" problem rather than a technical one :) )
